.github/workflows/autoreview.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        php-version: ['7.4']
```

Comment:
7.4 is still supported until 28 of November, 2021, and it's used [by ~60% of Infection users](https://github.com/infection/infection/blob/master/composer.json#L45), I think we should not get rid of it now.

Or, alternatively, if we want to get rid (personally, I'm ok with that) - we should bump the version in composer.json to ^8.0, here

https://github.com/infection/infection/blob/master/composer.json#L45

Reply:
reverted